### PR TITLE
Stripe Upgrade

### PR DESCRIPTION
**Pull request by stephencelis**
Taking this one to the finish line...

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -31,6 +31,16 @@ bootstrap-private:
...
 
+/usr/local/bin/stripe:
+	brew install stripe/stripe-cli/stripe
+
+stripe-listen: /usr/local/bin/stripe
+	stripe listen \
+		--events invoice.payment_failed,invoice.payment_succeeded,customer.subscription.deleted \
+		--forward-to localhost:8080/webhooks/stripe \
+		--latest \
+		--print-json
```
Comment (stephencelis) [inline Makefile]: lil make helper for web hooks

File: Sources/Stripe/Model.swift [hunk 2]
```diff
@@ -370,30 +368,26 @@ public struct Plan: Codable, Equatable {
...
     self.metadata = metadata
     self.nickname = nickname
-    self.statementDescriptor = statementDescriptor
     self.tiers = tiers
   }
 
   public func amount(for quantity: Int) -> Cents<Int> {
-    let amount = self.amount
-      ?? (self.tiers ?? []).first(where: { $0.upTo.map { quantity < $0 } ?? true })!.amount
+    // TODO: what to do with force unwrap?
```
Comment (stephencelis) [inline Sources/Stripe/Model.swift]: - [ ] TODO before merge

File: Tests/PointFreeTests/StripeWebhooksTests.swift [hunk 3]
```diff
@@ -23,111 +23,136 @@ final class StripeWebhooksTests: TestCase {
...
+  "request": {
+    "id": null,
+    "idempotency_key": null
+  },
+  "type": "invoice.payment_failed"
 }
 """
 
-    _ = try Stripe.jsonDecoder.decode(Stripe.Event<Stripe.Invoice>.self, from: Data(json.utf8))
+    _ = try! Stripe.jsonDecoder.decode(Stripe.Event<Stripe.Invoice>.self, from: Data(json.utf8))
```
Comment (stephencelis) [inline Tests/PointFreeTests/StripeWebhooksTests.swift]:
```suggestion
    _ = try Stripe.jsonDecoder.decode(Stripe.Event<Stripe.Invoice>.self, from: Data(json.utf8))
```

File: Makefile (hunk 1 above)
Comment (mbrandonw) [reply Makefile]: nice!

**Pull request merged**